inference-engine/src/mkldnn_plugin/nodes/mkldnn_split_node.cpp

```diff
@@ -170,9 +171,9 @@ void MKLDNNSplitNode::initSupportedPrimitiveDescriptors() {
     for (auto refPdIndex : pdIndexesToReuse) {
         const auto& refConfig = supportedPrimitiveDescriptors[refPdIndex].getConfig();
         auto config = refConfig;
-        const auto inBlockingDesc = refConfig.inConfs[0].desc->as<BlockedMemoryDesc>();
-        const auto& order = inBlockingDesc->getOrder();
-        const auto& blkDims = inBlockingDesc->getBlockDims();
+        const auto inBlockingDesc = MemoryDescUtils::convertToBlockedDescriptor(*refConfig.inConfs[0].desc);
+        const auto& order = inBlockingDesc.getOrder();
+        const auto& blkDims = inBlockingDesc.getBlockDims();
```

Comment:
We do not need such conversion in the `initSupportedPrimitiveDescriptors` stage. Please revert the changes in this function.